src/runtime.js

```diff
@@ -71,6 +70,8 @@ import {waitForBody} from './dom';
 import * as config from './config';
 
 initLogConstructor();
+dev();
+user();
```

Comment:
@mkhatib to answer your question though, yes those other main binaries call initLogConstructor